scripts/components/Interactions/OpenContent.js

```diff
@@ -224,9 +224,18 @@ export default class OpenContent extends React.Component {
       this.state.startMousePos,
       this.state.startMidPoint
     );
+  
```

Comment:
The maximum size will be the same in every call to `onMouseMove`, so why not set in the constructor and then just use the value? (a little nitpicking, yes)